jme3-core/src/test/java/com/jme3/scene/mesh/MeshTest.java

```diff
@@ -0,0 +1,16 @@
+package com.jme3.scene.mesh;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import com.jme3.scene.Mesh;
+
+public class MeshTest {
+       @Test
+    public void testEmptyMesh() {
+        final Mesh mesh = new Mesh();
+
+        assertEquals(-1, mesh.getVertexCount());;
```

Comment:
double semicolon ';;'  at the end of the instruction

Reply:
Changed it! Thank you for the Feedback!